src/metabase/lib/binning.cljc

```diff
@@ -69,54 +71,49 @@
     x]
    (available-binning-strategies-method query stage-number x)))
 
-(defn default-auto-bin
+(mu/defn default-auto-bin :- ::lib.schema.binning/binning-option
   "Returns the basic auto-binning strategy.
 
   Public because it's used directly by some drill-thrus."
   []
-  {:display-name (i18n/tru "Auto bin")
+  {:lib/type     :option/binning
+   :display-name (i18n/tru "Auto bin")
    :default      true
    :mbql         {:strategy :default}})
 
 (defn- with-binning-option-type [m]
   (assoc m :lib/type :option/binning))
 
-(def ^:private *numeric-binning-strategies
-  (delay (mapv with-binning-option-type
```

Comment:
Using a delay here is broken because it will only call `i18n/tru` when the delay is dereffed the first time, it will be broken if the user changes their language... the performance impact of recalculating i18n strings for binning options is not important enough to try to optimize things this way anyway.